packages/ibm-products/src/components/InlineTip/InlineTipButton.tsx

```diff
@@ -22,11 +22,24 @@ const componentName = 'InlineTipButton';
 
 // NOTE: the component SCSS is not imported here: it is rolled up separately.
 
+interface InlineTipButtonProps {
+  /**
+   * Provide an optional class to be applied to the containing node.
+   */
+  className?: string;
+  /**
+   * Provide the contents of the InlineTip.
+   */
+  children: ReactNode;
+}
 /**
  * This is a standard Carbon button, styled specifically for use inside InlineTip.
  */
 export let InlineTipButton = React.forwardRef(
-  ({ children, className, ...rest }, ref) => {
+  (
+    { children, className, ...rest }: PropsWithChildren<InlineTipButtonProps>,
+    ref: ForwardedRef<HTMLDivElement>
```

Comment:
```suggestion
    ref: ForwardedRef<HTMLButtonElement>
```

Reply:
Thanks @matthewgallo.
I've updated this.